### PR TITLE
Remote clion instructions

### PR DESCRIPTION
**Pull request by nicholasphair**

### CODE REVIEW

File: docs/clion_thin_client.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+[JetBrains Gateway][5] can be installed as a standalone app. It also comes part
+of any JetBrains IDE distribution. That is, if you have CLion installed
+already, you need not install the gateway. The instructions will continue with
+the JetBrains Gateway standalone app but are nearly identical, and have been
+tested with CLion.  
+
+First, launch the JetBrains Gateway application. From the "All Providers" page
+select "New Connection" under "SSH Connection".
+
+\image html gateway_all_provider_new_connection.png
```
Comment (Saket-Upadhyay) [inline docs/clion_thin_client.md:108]: This is not showing up in GitHub as an image, maybe try - ``

This is the case with all \image tags

Comment (Saket-Upadhyay) [reply docs/clion_thin_client.md:108]: Otherwise looks good

**Review by Saket-Upadhyay — CHANGES_REQUESTED** [review]

File: docs/clion_thin_client.md (hunk 1 above)
Comment (nicholasphair) [reply docs/clion_thin_client.md]: oh thats a doxygen tag, I wasn't thinking about viewing from gh, good catch. Fixed.

**Review by Saket-Upadhyay — APPROVED** [review]
Looks good.

**Pull request merged**